.github/workflows/check-url-changes.yml

```
@@ -0,0 +1,72 @@
+name: Check Documentation URL Changes
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check-url-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Identify deleted and renamed files
+        run: |
+          # Store deleted markdown files
```

Just for consistency
```suggestion
          # Store deleted files
```